src/components/locked-progress/locked-progress.tsx

```diff
@@ -38,9 +38,8 @@ export const LockedProgress = ({
   return (
     <div className="tranche-item__progress">
       <div
-        className={`tranche-item__progress-bar ${
-          light ? "tranche-item__progress-bar--light" : ""
-        }`}
+        className={`tranche-item__progress-bar ${light ? "tranche-item__progress-bar--light" : ""
```

Comment:
chore (blocking): bunch of pointless whitespace changes. Can you ensure that prettier is installed as an extension in your IDE please.

Also can you revert these changes and this is the incorrect formatting.

Reply:
done